### DISCUSSION TITLE
Icons dont load on qutebrowser

### DISCUSSION THREAD

**Discussion by tjulioh** [qutebrowser/qutebrowser#5561] [Q&A] [answered]
**Version info**:
qutebrowser v1.13.0

**Description**
Why icons dont load? after developed, i test and its occurred in qutebrowser.

**How to reproduce**
[Example](https://pasteboard.co/JfNeJYd.png)

In Chrome Browser: right side and correct;

Code in qutebrowser:  left side image and incorrect;

**Comment by The-Compiler** [comment]
Here's the image inline:

![image](https://user-images.githubusercontent.com/625793/86448775-8a469380-bd17-11ea-9c2a-9e997471cb3a.png)

I have no idea off-hand - is this a website I can visit to reproduce the issue? Does it happen when you start with `--temp-basedir` as well? Which backend and Qt version are you on?

**Comment by tjulioh** [comment]
I trying to  reproduce this on website opened

More icons dont load, same on --temp-basedir :

![icons](https://user-images.githubusercontent.com/62679312/86466969-f40c7080-bd0a-11ea-847c-f008773b9fee.png)

**Version info:**
qutebrowser v1.13.0
Git commit: 74ba8f3dc-dirty on HEAD (2020-06-26 08:31:20 +0200)
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.0
 
CPython: 3.7.7
PyQt: 5.15.0
 
sip: 5.3.0
colorama: 0.4.3
pypeg2: 2.15
jinja2: 2.11.2
pygments: 2.6.1
yaml: 5.3.1
cssutils: 1.0.2 $Id$
attr: 19.3.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: no
PyQt5.QtWebKitWidgets: no
pdf.js: 2.4.456 (bundled)
sqlite: 3.31.1
QtNetwork SSL: OpenSSL 1.1.1b  26 Feb 2019
 
Style: QWindowsVistaStyle
Platform plugin: windows
OpenGL: Intel, 4.0.0 - Build 9.17.10.3040
Platform: Windows-7-6.1.7601-SP1, 64bit
Frozen: True
Imported from C:\Program Files\qutebrowser\qutebrowser
Using Python from C:\Program Files\qutebrowser\qutebrowser.exe
Qt library executable path: C:/Program Files/qutebrowser/PyQt5/Qt/bin, data path: C:/Program Files/qutebrowser/PyQt5/Qt
OS Version: 7, 6.1.7601, SP1,

**Reply by The-Compiler** [reply]
Do you see anything interesting if you use `wi` to open the inspector and look at the console tab there?

**Reply by tjulioh** [reply]
![position](https://user-images.githubusercontent.com/62679312/86482862-29c05200-bd29-11ea-86e0-a6453c01af02.png)


Maybe the position? idk why, but only this is different, the styles are identical

**Reply by The-Compiler** [reply]
That's not the "console" tab though :wink:

**Reply by tjulioh** [reply]
Nothing in console, sry.

**Reply by The-Compiler** [reply]
I think you have the same issue as https://github.com/qutebrowser/qutebrowser/issues/5494

Can you also try https://lilpea.github.io/qutewebfonts/ and show the "Rendered Fonts" part of the inspector, like here:

![image](https://user-images.githubusercontent.com/625793/86581117-ed3a6380-bf7f-11ea-98e5-97c35e135e2b.png)

**Reply by tjulioh** [reply]
![font](https://user-images.githubusercontent.com/62679312/86588323-de44b880-bf61-11ea-9f15-44bc789adb6c.png)


The site is utilizing custom font.

**Comment by The-Compiler** [comment, marked as answer]
After our discussion above, I'm pretty sure this is the same issue as #5494. It's probably a bug in QtWebEngine, but I don't know why I can't reproduce it...

**Reply by tjulioh** [reply]
I will try reply on other version or linux, thks.

**Reply by The-Compiler** [reply]
Yep, that would be helpful! From what I've seen so far, it only seems to happen on Windows?

**Reply by tjulioh** [reply]
I tested on linx and work fine,

In Windows i try different versions of qute, the 1.12.0, 1.13.0 and 1.13.1 have this problem, the others dont, 1.11.1 worked.
